src/Util/ConsoleLogger.js

```diff
@@ -2,10 +2,16 @@ const chalk = require("chalk");
 const debug = require("debug")("Eleventy:Logger");
 const Readable = require("stream").Readable;
 const split = require("split");
+/** @typedef {import('stream').Readble Readable} */
```

Comment:
I'm a bit confused here. Is `Readble` a typo, or am I missing something?

Reply:
Obviously a typo m(